File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> hashMap = new HashMap<Integer, Integer>();
        for (int i = 0; i < nums.length; i++) {
            int req_num = target - nums[i];
            if (hashMap.containsKey(req_num)) {
                int[] arr = { hashMap.get(req_num), i };
                return arr;
            }
            hashMap.put(nums[i], i);
        }
        return null;
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        index1 = 0
        index2 = 1
        indices = []
        for i in range(len(nums)):
            for j in range(len(nums)):
                if i==j:
                    continue
                elif nums[i] + nums[j] == target:
                    index1 = i
                    index2 = j
                    indices.append(index1)
                    indices.append(index2)
                    break
            if len(indices) == 2:
                break
        return indices              

        

                    

        